<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>firstWindow</class>
 <widget class="QMainWindow" name="firstWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="height">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="width">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>height</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>width</string>
    </property>
   </widget>
   <widget class="QLabel" name="defaultlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Default configuration -&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="defaultBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Default</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom configuration</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Grid density </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="density">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QComboBox" name="box1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>T</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Z</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>L</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>J</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>I</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Brick 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Brick 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Brick 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Brick 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Brick 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Brick 6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>420</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Brick 7</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>T</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Z</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>L</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>J</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>I</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="box3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>T</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Z</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>L</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>J</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>I</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="box4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>T</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Z</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>L</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>J</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>I</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="box5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>390</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>T</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Z</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>L</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>J</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>I</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="box6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>T</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Z</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>L</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>J</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>I</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="box7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>T</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Z</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>L</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>J</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>I</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>25</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TETRIS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>448</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
